pkgs/top-level/python-packages.nix

```diff
@@ -3179,7 +3179,7 @@ in modules // {
     disabled = !isPy27;
 
     src = pkgs.fetchurl {
-      url = "http://cddb-py.sourceforge.net/${name}.tar.gz";
+      url = "mirror://sourceforge/${name}.tar.gz";
```

Comment:
http://downloads.sourceforge.net/project/cddb-py/cddb-py/1.4/CDDB-1.4.tar.gz

Add version as separate field and construct url from it.